pages.ja/windows/mkdir.md

```diff
@@ -0,0 +1,12 @@
+# mkdir
+
+> ディレクトリを作成します
+> 詳しくはこちら: <https://docs.microsoft.com/windows-server/administration/windows-commands/mkdir>.
+
+- ディレクトリを作成します:
+
+`mkdir {{directory_name}}`
+
+- ネストされたディレクトリツリーを再帰的に作成します:
```

Comment:
gTranslate show a possible typo in this example description.

Reply:
@navarroaxel 
May I know what is the typo that you have gotten from the gTranslate? This is the output that I get by retranslating it back to English and it looks fine:

![image](https://user-images.githubusercontent.com/12297298/95486293-699c7280-09c5-11eb-9d77-fdb8937f3de5.png)

Reply:
If I remove the `-`, gTranslate removes the `--C` :man_shrugging: but the render of the `.md` file translates OK.
![Screenshot_20201008_133652](https://user-images.githubusercontent.com/1536740/95487934-56f54f80-096b-11eb-9f8d-5a34f104729e.png)

Reply:
That's interesting.....not sure why the - broke it.